atomate/vasp/workflows/base/library/neb.yaml

```diff
@@ -0,0 +1,53 @@
+# A standard CI-NEB calculation
```

Comment:
@HanmeiTang If pymatgen-diffusion is now a dependency, you will need to add it in requirements.txt